app/models/custom_button.rb

```diff
@@ -13,6 +13,11 @@ class CustomButton < ApplicationRecord
   include UuidMixin
   acts_as_miq_set_member
 
+  TYPES = { "default"          => "Default",
+            "ansible_playbook" => "Ansible Playbook"}.freeze
+
+  PLAYBOOK_METHOD = "Order_Ansible_Playbook".freeze
```

Comment:
Link to where these are used?

Reply:
https://github.com/ManageIQ/manageiq-ui-classic/pull/1972